ilastik/workflows/tracking/conservation/conservationTrackingWorkflow.py

```diff
@@ -107,9 +107,12 @@ def __init__( self, shell, headless, workflow_cmdline_args, project_creation_arg
         self.trackingApplet = ConservationTrackingApplet( workflow=self )
 
         self.default_export_filename = '{dataset_dir}/{nickname}-exported_data.csv'
-        self.dataExportApplet = TrackingBaseDataExportApplet(self, 
-                                                             "Tracking Result Export", 
-                                                             default_export_filename=self.default_export_filename)
+        self.dataExportApplet = TrackingBaseDataExportApplet(
+            self,
+            "Tracking Result Export",
+            default_export_filename=self.default_export_filename,
+            doPluginExport=self._doPluginExport,
```

Comment:
Since `_doPluginExport` is not used anywhere else, doesn't use the local state and is very simple, maybe define right it here as a local function? Ditto for the other definitions.

Reply:
I prefer not to define functions inside constructors. Hard to find usages and extract in the future(prone to transform into closures).